### PR TITLE
feat: enable reset button

### LINKED ISSUE #56: [CHORE] Enable Seed Parsing for Resetting
When `SEED_EVERY_LESSON` flag is `true`, seed should be used, on load of next lesson.

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #56

### CODE REVIEW

**Review by moT01 — COMMENTED** [review]
Couple things I noticed when giving this a quick test...

I was getting an error from the `mkdir` command:
```
Error: Error: Command failed: mkdir test
mkdir: cannot create directory 'test': File exists
```
And the server would crash. I had issues like this with the RDB stuff. Not quite what the best method here is. Check if it exists first and then don't create it? Delete it and recreate it? If the reset, resets `git` to the beginning of the project - then the existence should 🤞  be predictable. Not sure if that happens or not. 

I think we could add a setting in the `project.json` to enable/disable this on a per project basis. Not sure if it's entirely necessary - but I'm thinking that when we get this PR in, the existing projects won't have this functionality yet. So we could write it all up and then enable it when it's ready.

Pretty minor, but it seemed to not work when I tried reset more than once in a row. e.g. click reset -> works, click it again without doing anything else -> didn't work.

**Comment by ShaunSHamilton** [comment]
Should all be fixed now.

> Pretty minor, but it seemed to not work when I tried reset more than once in a row. e.g. click reset -> works, click it again without doing anything else -> didn't work.

Not minor at all. I forgot to send the debounce response from the server. So, the client refused to send subsequent messages.

File: .freeCodeCamp/tooling/lesson.js [hunk 1]
```diff
@@ -54,13 +54,15 @@ async function runLesson(ws, projectDashedName) {
...
   const seed = getLessonSeed(lesson);
-  const isForce = isForceFlag(seed);
-  // force flag overrides seed flag
-  if (
-    (project.seedEveryLesson && !isForce) ||
-    (!project.seedEveryLesson && isForce)
-  ) {
-    await seedLesson(ws, project, lessonNumber);
+  if (seed) {
+    const isForce = isForceFlag(seed);
```
Comment (moT01) [inline .freeCodeCamp/tooling/lesson.js:58]: What is this isForce for?

**Review by moT01 — APPROVED** [review]
I'm fine with getting this in if you're ready @ShaunSHamilton 🎉 Seems to be working good 👍

File: .freeCodeCamp/tooling/lesson.js (hunk 1 above)
Comment (ShaunSHamilton) [reply .freeCodeCamp/tooling/lesson.js:58]: Sometimes, I feel you do not listen to any of my demonstrations 😢 

You can use the `#### --force--` flag to override the behaviour of:
a) Lesson automatically seeds on next lesson - force stops this
b) Lesson never seeds on next lesson - force causes the seed to be run when a Camper lands on the next lesson.

**Pull request merged**